### DISCUSSION TITLE
Release 1.9.0 in september

### DISCUSSION THREAD

**Discussion by jeromerobert** [pdfarranger/pdfarranger#693] [Announcements]
Dear all,

If no one objects, I would like to release 1.9.0 in the next few weeks, let's say in september.

It's a good time to test and translate the main branch.

@albanobattistella, @emintufan, @hahnavi, @hugok79, @juanma1980, @juxuanu, @mjkim0727, @milotype, @MichelePezza, @oscfdezdz, @ovari, @pavelzahradnik, @sicklylife-jp, @smarquespt, @sveinki, @Vistaus, @volkov0aa, @wcxu21, @yarons (not notified because of recent contributions: Ser82-png, Caesar)

Regards,

Jerome

Edit by @dreua : @kuroehanako changed to @mjkim0727 according to their request below.

**Comment by hugok79** [comment]
Portuguese(pt_PT) translation is done https://github.com/pdfarranger/pdfarranger/pull/695

**Comment by jeromerobert** [comment]
Release notes drafted.

**Reply by ovari** [reply]
Could you please advise the url to the draft release notes? Thank you

**Reply by jeromerobert** [reply]
The URL is only available to the PDF Arranger core team (that's the way github work). Here is a copy/past:

* Windows portable zip: `config.ini` is located in the same folder as `pdfarranger.exe` (#402)
* Many improvements and fixes in the icon view (#505, #506, #612, #629, #632, #637, #654, #670)
* Remember previous file chooser location (#508)
* Add close and open actions (#581, #593, #597)
* Increase I/O performance and responsiveness (#542, #623)
* Add option to set language in Windows (#578)
* Add support for pasting an image from clipboard (#675)
* Pan the view when pressing mouse wheel and moving mouse (#677)
* Add fullscreen mode (#665)
* Allow to print the document (#647)

**Comment by ovari** [comment]
Can you please add the new strings to the Hungarian file so we can translate them using a web browser? Thank you

**Reply by jeromerobert** [reply]
Done

**Reply by ovari** [reply]
Hungarian (magyar) https://github.com/pdfarranger/pdfarranger/pull/705

1. Does `msgid "Zoom _Fit"` mean `Zoom to _fit` (`_Mérethez igazítás`) or `Size to _Fit` (`Ill_eszkedő méret`)?
https://www.microsoft.com/en-us/language/Search?&searchTerm=Fit&langID=382&Source=true&productid=undefined

2. Do `msgid "% of width"` and `msgid "% of height"` have values before or after the string?
`% of &width:` https://www.microsoft.com/en-us/language/Search?&searchTerm=of%20width&langID=382&Source=true&productid=undefined

Thank you

**Comment by juxuanu** [comment]
Done https://github.com/pdfarranger/pdfarranger/pull/703

**Comment by jeromerobert** [comment]
For now, I am only regenerating the `.pot` file and leave you free to do what you want with the `.po`. Some translators would prefer me to update their `.po` as well. So I suggest to switch to this way of working for everyone. By updating all `.po` files I mean running this from time to time (particularly near to releases):

- `msgmerge -U` (merge from `.pot` to `.po`)
- `msgattrib --no-fuzzy --no-obsolete` (remove fuzzy and `#~`)

If you disagree, please speak up.

**Comment by Vistaus** [comment]
I don't mind. Merging the changes is easy enough, either with your instructions or through Poedit: Translation --> Update from POT file...

**Comment by jeromerobert** [comment]
I plan to release 1.9.1 on the next saturday.

**Reply by ovari** [reply]
Are there strings to translate?

**Reply by ovari** [reply]
If there are strings to translate, can you please add the English source strings to the hu.po file with blank Hungarian translations?

**Reply by jeromerobert** [reply]
No, there were no changes in strings to translate compared to 1.9.0.

**Comment by jeromerobert** [comment]
I plan to release a 1.9.2 soon so we have the latest 3 fixes in a release before we start merging more disruptive thinks like #760.

**Comment by mjkim0727** [comment]
@kuroehanako changed @mjkim0727

**Reply by dreua** [reply]
Thanks for making us aware! I see the user @kuroehanako no longer exists and edited the top post accordingly in case someone refers back to that.
